### PR TITLE
return empty response when subscription are running on the internal introspection service

### PR DESCRIPTION
**Pull request by temaEmelyan**
Please make sure you consider the following:

- [x] Add tests that use __typename in queries
- [x] Does this change work with all nadel transformations (rename, type rename, hydration, etc)? Add tests for this.
- [x] Is it worth using hints for this change in order to be able to enable a percentage rollout?
- [x] Do we need to add integration tests for this change in the graphql gateway?
- [x] Do we need a pollinator check for this?

### CODE REVIEW

File: test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+serviceCalls: [ ]
+# language=JSON
+response: |-
+  {
+    "errors": [
+      {
+        "locations": [],
+        "message": "null",
+        "extensions": {
+          "classification": "ValidationError"
```
Comment (temaEmelyan) [inline test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml:53]: this error is coming from a "remove field" transform that we use for tests. I activated this transform with `onCommentUpdated(id: ID): Comment @toBeDeleted` directive in the overall schema

Comment (temaEmelyan) [reply test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml:53]: previously we would blow up with `your data fetcher must return a publisher ...` kind of an error

File: lib/src/main/java/graphql/nadel/engine/blueprint/IntrospectionService.kt [hunk 2]
```diff
@@ -40,6 +41,9 @@ open class NadelDefaultIntrospectionRunner(schema: GraphQLSchema) : ServiceExecu
         .build()
 
     override fun execute(serviceExecutionParameters: ServiceExecutionParameters): CompletableFuture<ServiceExecutionResult> {
+        if (serviceExecutionParameters.operationDefinition.operation == OperationDefinition.Operation.SUBSCRIPTION) {
+            return CompletableFuture.completedFuture(NadelServiceExecutionResultImpl())
```
Comment (russellyou) [inline lib/src/main/java/graphql/nadel/engine/blueprint/IntrospectionService.kt:45]: So just empty result?

**Review by russellyou — APPROVED** [review]

**Review by bbakerman — DISMISSED** [review]

File: test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml (hunk 1 above)
Comment (gnawf) [inline test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml:53]: So this is an error from the introspection service? Why is the error message null?

Comment (temaEmelyan) [reply test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml:53]: nah, this error is coming from the transform, look at this comment https://github.com/atlassian-labs/nadel/pull/649/files/5774cd1580b7dcaeaecb6b6b7d46bdea52b078eb#r1870548215

Comment (gnawf) [reply test/src/test/resources/fixtures/field removed/top-level-field-is-removed-for-a-subscription-with-namespaced-field.yml]: Ah I see

**Review by gnawf — APPROVED** [review]

**Pull request merged**